R/FilterStateChoices.R

```diff
@@ -494,7 +520,7 @@ ChoicesFilterState <- R6::R6Class( # nolint
             )
           } else {
             private$observers$selection <- observeEvent(
-              ignoreNULL = FALSE, # it's possible that nothing is selected
+              ignoreNULL = private$init_null, # init_null toggle for multiple var.
```

Comment:
@kartikeyakirar isn't this better?

```suggestion
              ignoreNULL = !private$multiple,
```

Reply:
Indeed, @gogonzo, I had a similar idea, and it appears to be cleaner as well. However, I initially thought it might be confusing while reviewing the code, which led me to create the "init_null" variable. Nevertheless, I now believe that this approach is cleaner.

Reply:
updated it and removed init_null